flink-python/pyflink/table/table_config.py

```diff
@@ -44,18 +47,55 @@ def parallelism(self):
     def parallelism(self, parallelism):
         self._parallelism = parallelism
 
+    @property
+    def timezone(self):
+        return self._j_table_config.getTimeZone().getID()
```

Comment:
Takes the Python timezone object as the return type?

Reply:
The reason for keeping current implementation is the same as above.

Reply:
OK.